boxcli/featureflag/feature.go

```diff
@@ -44,3 +44,7 @@ func (f *feature) Enabled() bool {
 	}
 	return f.enabled
 }
+
+func (f *feature) Disabled() bool {
```

Comment:
Why both `Enabled()` and `Disabled()`?

Reply:
I found it more clear in the caller code.

In my initial implementation, I missed the not operator in `!featureflag.<name>.Enabled()` and then switched to Disabled which is more clear.